.aws/src/main.ts

```diff
@@ -295,6 +295,15 @@ class RecommendationAPI extends TerraformStack {
                         ],
                         effect: 'Allow'
                     },
+                    {
```

Comment:
Would it be better to attach the managed policy `arn:aws:iam::996905175585:policy/PocketDataProductReadOnly`?

Reply:
I considered that, but `PocketDataProductReadOnly` is currently only available in our production account, and I like that RecommendationAPI is able to deploy to both Pocket-Dev and production.

Reply:
I like it, too—until and unless we make a feature decision in the future that requires Recs API to write data somewhere (for example if we end up needing to record what we sent in which order to which user such that we can more effectively investigate cases of misinformation recommendations in the future). 

NOT A NEAR TERM CONCERN. But wanted to mention it apropos of our prior discussions about metaflow, feature groups, and prefect that resulted in our decisions to stay prod-only with our data writes.